Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -446,8 +446,10 @@ bool AXObjectCache::isNodeVisible(Node* node) const
     }
 
     // We also need to consider aria hidden status.
-    if (!isNodeAriaVisible(*node))
-        return false;
+    if (auto* element = dynamicDowncast<Element>(node)) {
```

Comment:
We already do an `is<Element>(node)` at the top of this function, so this is redundant. I propose we add this to the top of the function, and avoid the need for a cast on this line completely:

```cpp
RefPtr element = dynamicDowncast<Element>(node);
if (!element)
    return false;
```

Reply:
I like that, definitely cleaner. Will make that change!